altair/utils/schemapi.py

```diff
@@ -564,11 +564,17 @@ def to_dict(self, validate=True, ignore=None, context=None):
             try:
                 self.validate(result)
             except jsonschema.ValidationError as err:
-                raise SchemaValidationError(self, err)
+                raise SchemaValidationError(self, err) from err
         return result
 
     def to_json(
-        self, validate=True, ignore=[], context={}, indent=2, sort_keys=True, **kwargs
+        self,
+        validate=True,
+        ignore=None,
```

Comment:
Can't comment in the docstring below but could you add an `(optional)` for ignore there same as for context?

Reply:
done